net.tangysoft.usenext.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <!--Created with jdAppStreamEdit 9.1-->
+  <id>net.tangysoft.usenext</id>
+  <name>UseNeXt</name>
+  <summary>Usenext Usenet reader</summary>
+  <developer id="net.tangysoft">
+    <name>Tangysoft</name>
+  </developer>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Is redistribution allowed under the license? If not this has to use extra-data.

Reply:
That's a very good question! Thanks for catching it.
I couldn't find anything that would allow redistribution.
I guess extra-data is the way to go.
Let me read up on how to do that.

Reply:
extra-data with unversioned URL is going to be problematic if the size or hash changes it will fail to install for people.